icecreamshop/src/main/java/com/summerschool/icecreamshop/service/ProductService.java

```diff
@@ -15,20 +15,25 @@
     @Autowired
     private ProductRepository productRepository;
 
-    public List<Product> getAllByTitle(String title) {
-        return productRepository.findByTitle(title);
-    }
+
 
     public Optional<Product> get(Long id)
     {
         return productRepository.findById(id);
     }
 
+    public Product add(Product product, Category category)
+    {
```

Comment:
pune acolada pe randul cu definirea metodei:  public Product add(Product product, Category category){    vezi ca ai problema asta in mai multe locuri